pages/osx/rubocop.md

```diff
@@ -0,0 +1,31 @@
+# rubocop
+
+> Lint Ruby files.
+
+- Check all files in the current directory:
+
+`rubocop`
+
+- Check specific files or directories:
+
+`rubocop {{path_1}} {{path_2}}`
```

Comment:
Change the path tokens to usual path syntax - `path/to/file`, `path/to/directory`